# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/base/test/test_bug1576154.html
# Commit: b72d1d6131cc
# Full Hash: b72d1d6131cce6845af6ffd6f83de7f993930961
# Author: Edgar Chen <echen@mozilla.com>
# Date: 2019-08-26 09:42:55
# Description:
#   Bug 1576154 - Check mSVGDocumentWrapper when trying to report use counters; r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43256
# ==============================================================================

diff -r 929421783c5b -r b72d1d6131cc dom/base/test/test_bug1576154.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/base/test/test_bug1576154.html	Sun Aug 25 22:45:33 2019 +0000
@@ -0,0 +1,36 @@
+<!DOCTYPE HTML>
+<html>
+<!--
+https://bugzilla.mozilla.org/show_bug.cgi?id=1576154
+-->
+<head>
+  <title>Test for Bug 1576154</title>
+  <script type="text/javascript" src="/MochiKit/MochiKit.js"></script>
+  <script src="/tests/SimpleTest/SimpleTest.js"></script>
+  <link rel="stylesheet" type="text/css" href="/tests/SimpleTest/test.css" />
+</head>
+<body>
+<a target="_blank" href="https://bugzilla.mozilla.org/show_bug.cgi?id=1576154">Mozilla Bug 1576154</a>
+<p id="display"></p>
+<!-- bug1576154.sjs returns an SVG image with HTTP error code 500. -->
+<img src="bug1576154.sjs">
+<pre id="test">
+<script class="testbody" type="text/javascript">
+
+/** Test for Bug 657191 **/
+SimpleTest.waitForExplicitFinish();
+
+window.onload = function() {
+  let img = document.querySelector("img");
+  img.src = "green.png";
+  img.onload = function() {
+    // As long as this doesn't crash, this test passes.
+    ok(true, "test passed");
+    SimpleTest.finish();
+  };
+}
+
+</script>
+</pre>
+</body>
+</html>